.versionbump.cfg

```diff
@@ -0,0 +1,16 @@
+[bumpversion]
```

Comment:
The name of this file should be `.bumpversion.cfg` and not `.versionbump.cfg`. Did you test this change?

Reply:
Indeed, I tested it now